### PR TITLE
Add Dumpling and Cheeseburger

### PR DESCRIPTION
**Pull request by Zed9K**

### CODE REVIEW

File: build_targets [hunk 1]
```diff
@@ -2,8 +2,10 @@
 # <device> <build_type> <auto-upload> <saveimages>
 a73xq userdebug yes [recovery.img]
 caiman userdebug yes [boot.img, dtbo.img, init_boot.img, vendor_boot.img, vendor_kernel_boot.img]
+cheeseburger user yes [boot.img, recovery.img]
```
Comment (gwolf2u) [inline build_targets]: you sure you need both?

File: build_targets [hunk 2]
```diff
@@ -2,8 +2,10 @@
...
 comet userdebug yes [boot.img, dtbo.img, init_boot.img, vendor_boot.img, vendor_kernel_boot.img]
 courbet user yes [recovery.img]
+dumpling user yes [boot.img, recovery.img]
```
Comment (gwolf2u) [inline build_targets]: same here

**Comment by Zed9K** [comment]
honestly I don't need any 
but i was like why not 
can I keep it blank ? 
or just do recovery only

**Comment by gwolf2u** [comment]
can be blank, but suggest to use crDroid recovery if possible

**Pull request closed without merging**